src/test/java/org/ccci/obiee/client/rowmap/RowmapIntegrationTest.java

```diff
@@ -72,12 +72,29 @@ public void testRetrieveWithDesignationParameter() throws Exception
         query.withSelection(params);
         List<SaiDonationRow> rows = query.getResultList();
 
-        assertThat(rows.size(), greaterThan(0));
+        assertThat(rows.size(), greaterThan(1000));
```

Comment:
Enough, I think. I don't think the DW will ever lose (many) donations for a certain designation. The actual count is currently 8000ish.